### PR TITLE
Internal: Skip required workflow in GitHub CI [ED-5930]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -48,3 +48,17 @@ jobs:
         run: composer run test
       - name: Run Test - Multisite
         run: WP_MULTISITE=1 composer run test
+
+  test-result:
+    if: ${{ always() }}
```
Comment (matipojo) [inline .github/workflows/phpunit.yml:53]: Why not
```suggestion
```

File: .github/workflows/phpunit.yml [hunk 2]
```diff
@@ -48,3 +48,17 @@ jobs:
...
+    runs-on: ubuntu-18.04
+    name: Test Results
```
Comment (matipojo) [inline .github/workflows/phpunit.yml:55]:
```suggestion
    name: PHPUnit - Test Results
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: .github/workflows/phpunit.yml (hunk 2 above)
Comment (Nevoss) [reply .github/workflows/phpunit.yml]: Done

File: .github/workflows/phpunit.yml [hunk 3]
```diff
@@ -48,3 +48,17 @@ jobs:
         run: composer run test
       - name: Run Test - Multisite
         run: WP_MULTISITE=1 composer run test
+
+  test-result:
+    if: ${{ always() }}
+    runs-on: ubuntu-18.04
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply .github/workflows/phpunit.yml]: Done

File: .github/workflows/phpunit.yml (hunk 1 above)
Comment (Nevoss) [reply .github/workflows/phpunit.yml]: Added a comment

**Pull request merged**